designate.md

```diff
@@ -0,0 +1,28 @@
+Deploying Designate
+-------------------
+
+Designate provides SUSE OpenStack Cloud, DNS as a service. It allows creating and propagating zones
+and records over the network using pools of DNS server. From an Admin's perspective there is not
+much to configure as the default for deployment are well in place.
```

Comment:
I believe there are steps needed to enable the neutron integration. Is the barclamp doing this automatically?

https://docs.openstack.org/designate/latest/contributor/integrations.html
https://docs.openstack.org/ocata/networking-guide/config-dns-int.html